actionview/lib/action_view/template/handlers/erb.rb

```diff
@@ -45,6 +45,9 @@ def call(template, source)
           # Always make sure we return a String in the default_internal
           erb.encode!
 
+          # Strip trailing newlines from the template if enabled
+          erb.chomp! if ActionView::Base.strip_trailing_newlines
```

Comment:
Now that this only affects ERB, maybe this should be a class attribute in this file on `ActionView::Template::Handlers::ERB`?

Reply:
In the event that this functionality gets implemented for other template handlers, I suppose we'd move it back to `ActionView::Base`, right?

This seems reasonable — I'll get it in and then squash those commits.